src/Donations/Endpoints/ListDonations.php

```diff
@@ -245,8 +250,8 @@ public function getTotalDonationsCount(): int
     protected function getWhereConditions(QueryBuilder $query): array
     {
         $search = $this->request->get_param('search');
-        $start = $this->request->get_param('start');
-        $end = $this->request->get_param('end');
+        $start = $this->maybeExpandDate($this->request->get_param('start'), 'before');
+        $end = $this->maybeExpandDate($this->request->get_param('end'), 'after');
```

Comment:
This is related [to this comment](https://github.com/impress-org/givewp/pull/8140/files#r2437179417).  From an API perspective we should make this simpler by update the `start` and `end` args to use the date-time format (no need to have a custom sanitizer after that) and just let the front-end send dates along.